integration_test/init_test.go

```diff
@@ -8,9 +8,17 @@ import (
 )
 
 const (
-	xplaGeneralGasLimit int64 = 240000
-	xplaCodeGasLimit    int64 = 5000000
-	xplaGasPrice              = "8500000000"
+	xplaLowGasLimit             int64 = 24
+	xplaGeneralGasLimit         int64 = 240000
+	xplaXATPTransferGasLimit    int64 = 600000
+	xplaProposalGasLimit        int64 = 500000
+	xplaCreatePairGasLimit      int64 = 650000
+	xplaPairGasLimit            int64 = 800000
+	xplaCodeGasLimit            int64 = 6000000
+	xplaPairInstantiateGasLimit int64 = 8000000
+
+	xplaGasPrice = "8500000000"
+	tknGasPrice  = "8.5"
```

Comment:
The current pool ratio is 1 (Xpla):10 (tokens). I think tknGasPrice is 0.085uTKN.

Reply:
https://github.com/xpladev/xpla/pull/60/commits/3d58e61983b11b13a91450924cc22f670d2669d8